src/Events/EventSeverity/index.js

```diff
@@ -77,9 +136,9 @@ class EventSeverities extends Component {
   /**
    * @function
    * @name handleEdit
-   * @description Handle on Edit action for list item
+   * @description Handle on Edit severity for list item
    *
-   * @param {object} eventSeverity event severity to be edited
+   * @param {object} eventSeverity eventSeverity to be edited
```

Comment:
Aside from variable names all other words are normal, do not convert them to camel case 
e.g. `@param {object} eventSeverity Event severity to be edited`